nodes/src/nodes/tool_bland_ai/IInstance.py

```
@@ -0,0 +1,46 @@
+from rocketlib import IInstanceBase
+
+from .IGlobal import IGlobal
+
+
+class IInstance(IInstanceBase):
+    IGlobal: IGlobal
+
+    def invoke(self, param: Any) -> Any:  # noqa: ANN401
+        driver = getattr(self.IGlobal, "driver", None)
+        if driver is None:
+            raise RuntimeError("bland_ai: driver not initialized — check API key configuration")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)